cupy/cudnn.py

```diff
@@ -326,3 +328,14 @@ def add_tensor(handle, alpha, biasDesc, biasData, beta, srcDestDesc,
                srcDestData):
     cudnn.addTensor_v3(handle, alpha, biasDesc,
                        biasData, beta, srcDestDesc, srcDestData)
+
+
+def is_tensor_core_available(dtype):
+    global _compute_capability
+    if _compute_capability is None:
+        _compute_capability = int(device.Device().compute_capability)
```

Comment:
Please use `dict` that holds device id and compute_capability pair.
Or, use `util.memoize(for_each_device=True)`.

Reply:
Considering that both cuBLAS and cuDNN require caching of `compute capability` value, I think it is better to implement this in `device.pyx` to reduce duplication. What would you think on this?